docs/core/tools/dotnet-store.md

```diff
@@ -0,0 +1,78 @@
+---
+title: dotnet-store command - .NET Core CLI | Microsoft Docs
+description: The dotnet-store command stores the specified assemblies in the runtime package store.
+keywords: dotnet-store, dotnet-publish, CLI, CLI command, .NET Core
+author: beleroy
+manager: wpickett
+ms.date: 10/07/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+---
+
+#dotnet-store
+
+## Name
+
+`dotnet-store` - Stores the specified assemblies in the [runtime package store](../deploying/runtime-package-store.md).
+
+## Synopsis
+
+`dotnet store [--manifest] [--framework-version] [--output] [--working-dir] [--preserve-working-dir] [--skip-optimization] [-v|--verbosity] [-h|--help]`
+
+## Description
+
+`dotnet store` stores the specified assemblies in the [runtime package store](../deploying/runtime-package-store.md).
+
+By default, these will be optimized for the target runtime and framework.
+
+For more information, see the [runtime package store](../deploying/runtime-package-store.md) topic.
+
+## Options
+
+`-h|--help`
+
+Prints out a short help for the command.  
+
+`-m|--manifest <TARGET_MANIFEST>`
+
+The XML file, or a list of XML files, that contain the list of packages to be stored. 
```

Comment:
how do you specify the list of XML files? also, on the examples you're linking to a project file, so the XML files are or can be project files? And if you don't specify manifest, what happens?